internal/pkg/cli/env_init.go

```diff
@@ -461,22 +461,28 @@ https://aws.amazon.com/premiumsupport/knowledge-center/ecs-pull-container-api-er
 		if err != nil {
 			if err == selector.ErrSubnetsNotFound {
 				log.Errorf(`No existing public subnets were found in VPC %s. You can either:
```

Comment:
Can we constantize these messages since they're a bit complex?

Reply:
Got rid of the complexity altogether! 😄